### PR TITLE
Adding Navbar

### PR DESCRIPTION
**Pull request by AbdulBasit0044**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <Navbar />
+        <Routes>
+          <Route exact path="/" element={<Homepage />} />
+          <Route exact path="/about" element={<About />} />
```
Comment (omerzn) [inline src/App.js]: dont need to pass `exact` here and the next line. Please remove it.

File: src/index.css [hunk 2]
```diff
@@ -1,13 +1,105 @@
...
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: "Quicksand";
+  /* color: #333; */
+}
+.navbar {
```
Comment (omerzn) [inline src/index.css]: don't add styling to index.css. Only add theme related styling here. Component specific styling should be done like we discussed: styled components. We can go over it again on call.

**Review by omerzn — CHANGES_REQUESTED** [review]
We need to change the structure a bit. Move each component (homepage, navbar and create) file except the required files (like app.js) to `src/components`. You can create a folder for homepage inside `src/components` and bundle everything related to the homepage component inside it. Likewise for every other component that you have created or will create in the future. Please make these changes and request and re-review.

File: src/App.js (hunk 1 above)
Comment (AbdulBasit0044) [reply src/App.js]: Removed

File: src/index.css (hunk 2 above)
Comment (AbdulBasit0044) [reply src/index.css]: Done

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,22 @@
...
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Homepage />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/create" element={<Create />} />
```
Comment (omerzn) [inline src/App.js]:
```suggestion
          <Route path="about" element={<About />} />
          <Route path="create" element={<Create />} />
```
RR 6 implementation

**Review by omerzn — CHANGES_REQUESTED** [review]
Please make the suggested change and also read my message on slack.

File: src/App.js (hunk 3 above)
Comment (AbdulBasit0044) [reply src/App.js]: Done

**Review by omerzn — APPROVED** [review]
Great job @AbdulBasit0044

**Pull request merged**